kotest-assertions/src/commonMain/kotlin/io/kotest/properties/shrinking/IntShrinker.kt

```diff
@@ -3,14 +3,27 @@ package io.kotest.properties.shrinking
 import kotlin.math.abs
 
 object IntShrinker : Shrinker<Int> {
-  override fun shrink(failure: Int): List<Int> =
+   override fun shrink(failure: Int): List<Int> =
       when (failure) {
-        0 -> emptyList()
-        1, -1 -> listOf(0)
-        else -> {
-          val a = listOf(0, 1, -1, abs(failure), failure / 3, failure / 2, failure * 2 / 3)
-          val b = (1..5).map { failure - it }.reversed().filter { it > 0 }
-          (a + b).distinct().filterNot { it == failure }
-        }
+         0 -> emptyList()
+         1, -1 -> listOf(0)
+         else -> {
+            val a = listOf(0, 1, -1, abs(failure), failure / 3, failure / 2, failure * 2 / 3)
+            val b = (1..5).map { failure - it }.reversed().filter { it > 0 }
+            (a + b).distinct().filterNot { it == failure }
+         }
+      }
+}
+
+object LongShrinker : Shrinker<Long> {
```

Comment:
I'm unsure if LongShrinker should be at IntShrinker file

Reply:
That's the old file.